src/components/container/container.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function Container(props) {
+	const param = props;
+	return (
+		<div className="container">
+			{param.children}
```

Comment:
I don't quite understand why you don't just use props.children?)

Reply:
Removed unnecessary constant.